content/root/best-practice-guide.textile

```diff
@@ -58,7 +58,7 @@ The main factors are summarized here:
 
 * If you want to maintain a persistent connection with Ably's Realtime platform in order to subscribe to messages sent on a channel, or be notified whenever the presence set changes, you need to use the "realtime library":/realtime.
 * If you just want to publish data to the platform and don't need to subscribe or manage channel attachment, you can the "REST library":/rest.
-* If you want to publish at very high rates or with the lowest possible realtime latencies, "Ably's realtime library would be best as it can publish at higher rates and lower latencies than is possible with the REST library due to message pipelining":https://faqs.ably.com/should-i-use-the-rest-or-realtime-library.
+* If you want to publish at very high rates or with the lowest possible realtime latencies, "Ably's realtime library would be best as it can publish at higher rates and lower latencies than is possible with the REST library due to message pipelining":https://ably.com/documentation/best-practice-guide#rest-or-realtime.
```

Comment:
I don't think this needs to be a link as it's linking to itself.

Reply:
Agreed, I've just removed the link totally now.